router.go

```diff
@@ -0,0 +1,69 @@
+package serverfull
+
+import (
+	"net/http"
+
+	"github.com/asecurityteam/runhttp"
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+)
+
+// RouterConfig is used to alter the behavior of the default router
+// and the HTTP endpoint handlers that it manages.
+type RouterConfig struct {
+	// HealthCheck defines the route on which the service will respond
+	// with automatic 200s. This is here to integrate with systems that
+	// poll for liveliness. The default value is /healthcheck
+	HealthCheck string
+
+	// Fetcher is the Lambda function loader that will
+	// be used by the runtime. There is no default for this value.
```

Comment:
question: if we have an implementation of this in the project, why not make it the default?

Reply:
Event the built-in "static" version of the fetcher requires configuration of some kind. In that case it requires a user provided map of names to functions. There's no way to make that the default without tightly coupling to that implementation.